presto-spi/src/main/java/com/facebook/presto/spi/type/SqlTime.java

```diff
@@ -19,30 +19,47 @@
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
 import java.util.Objects;
+import java.util.Optional;
+
+import static com.facebook.presto.spi.type.TimeZoneKey.UTC_KEY;
 
 public final class SqlTime
```

Comment:
While at it... consider adding a javadoc, like there is in `SqlTimeWithTimeZone`. E.g. you might likewise explain range of possible values for the `millisUtc` field.